README.md

```diff
@@ -1,112 +1,137 @@
+```
+mpiexec -np <nproc> ./fluid-cpp/build/FluidSolver precice-config.xml N tau kappa -parallel
+```
+and
+```
+mpiexec -np <nproc> ./solid-cpp/build/SolidSolver precice-config.xml N -parallel
+```
+A working known combination for the input parameters is N=100, tau = 0.01, kappa = 100. Other parameters like the simulation's end time, you can modify them in the precice-config.xml.
 
-2. Make the tutorial:
-   ```bash
-   $ make all
-   ```
+### python
 
-3. After successful compilation, you can now launch preset configuration by calling the `Allrun` script located in the current folder:
-   ```bash
-   $ ./Allrun
-   ```
-   Results will be stored as .vtk files in the `cxx/Postproc` folder.
+Open two separate terminals and start each participant by calling the respective run script. Only serial run is possible:
+
+```
+python3 ./fluid-python/FluidSolver.py precice-config.xml 
+```
+and
+```
+python3 ./solid-python/SolidSolver.py precice-config.xml 
+```
+Parameters such as N can be modified directly at the `FluidSolver.py` and at the `SolidSolver.py`. The parameters must be consistent between the different solvers and participants. 
+
+**Optional:** Visualization and video output of the fluid participant can be triggered via the options `--enable-plot` and `--write-video` of `FluidSolver.py`. To generate .vtk files during execution, you need to add the flag `--write-vtk`.
+
+## Post-processing
+
+The `Postproc/` folder contains the .vtk files resulting from the execution, which you can visualize using eg. paraview. Alternatively you can visualize the results with the provided `Postproc/fluid.py` script:
 
-4. To quickly clean the folder of log files and results from previous runs, execute `Allclean`:
-   ```bash
-   $ ./Allclean
-   ```
-   
-**Optional:** You can visualize the results with the `Postproc/fluid.py` script:
 ```bash
-$ python Postproc/fluid.py <quantity> Postproc/<prefix>
+$ python3 Postproc/fluid.py <quantity> Postproc/<prefix>
 ```
 Note the required arguments specifying which quantity to plot (`pressure`, `velocity` or `diameter`) and a name prefix for the target vtk files.
+
 For example, to plot the diameter using the default prefix for vtk files, we execute:
 ```bash
-$ python Postproc/fluid.py diameter Postproc/out_fluid_
+$ python3 Postproc/fluid.py diameter Postproc/out_fluid_
 ```
-An image of this diameter plot can be found in the `cxx/example` folder.
+![FSI3 setup](images/python.png)
 
-**Alternative:**: If you wish to run the parallel versions of each solver, run the `Allrun_parallel` script instead. Note that no vtk output is generated for this solver configuration!
+If you run the case in parallel, you can visualize the results calculated by one rank (eg. rank 0) as follows:
+
+```bash
+$ python3 Postproc/fluid.py diameter Postproc/out_fluid_
+```
+
+An image of this diameter plot can be found in the `/images` folder.
```

Comment:
Maybe we could also enable the watchpoint in the `precice-config.xml` and add the picture here? The picture is already in `images/`.

We could also delete the old `images/diameter.png`.